### PR TITLE
Adding legens to export functionality

### LINKED ISSUE #5206: [Suggestion] Add legends to export functionality
As visible in this screenshot this map displays one absolute value by bubble size (as with layer wizard: bubble) and one relative value by color (as with layer wizard: chloropleth): 

screenshot from 2015-08-24 17 57 01 

The export function however offers no option to include this legend: 
export function no legend

The exported image displays no legend: 
ejd3sgdl

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2015-10-14 at 16 26 14
download-2

Fixes #5206 
REVIEWER: @javierarce

### CODE REVIEW

File: lib/assets/javascripts/cartodb/table/mapview.js [hunk 1]
```diff
@@ -482,11 +482,9 @@ cdb.admin.MapTab = cdb.core.View.extend({
     }
 
     var showLegends = this.map.get("legends");
-    showLegends ? this.$el.removeClass("hide_legends") : this.$el.addClass("hide_legends");
 
     this.map.bind("change:legends", function() {
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/mapview.js]: `showLegends` is not really used anymore, right? I think the whole binding could be deleted.

Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/mapview.js]: True, thanks!

**Comment by javierarce** [comment]
:+1:

**Comment by xavijam** [comment]
I guess you are interested in this one too @mbforr ;)

**Comment by mbforr** [comment]
Amazing! Thanks @xavijam

**Pull request merged**